fund_store/config/fund_loader_config/FAB/gbrf_r1.py

```diff
@@ -0,0 +1,87 @@
+        "description_json": {
+            "en": "To ensure that local authorities are well-equipped and supported to implement the changes in the National Planning Policy Framework (NPPF), the Government has announced funding to provide additional support to local authorities that will need to undertake a Green Belt review, in the form of a direct financial contribution towards their costs. Local authorities that need to undertake a Green Belt review to accommodate an increase in their local housing need as a result of the NPPF changes will be able to submit an Expression of Interest (EOI) for a share of the Fund, up to a maximum of £70k per local authority. The funding will provide a direct financial contribution to eligible local authorities to enable them to complete their Green Belt review.\u202f\u202fSuccessful local authorities will be notified that they will receive funding on or around 10 February, and payments are expected to be made by 19 February 2025. ",
+            "cy": "",
+        },
+    },
+    "round_config": {
+        "id": "e480f03f-e3e0-4bd0-9026-dfed52cc3982",
+        "fund_id": "f97e3930-ab32-4353-84a6-3053d05382ae",
+        "short_name": "R1",
+        "opens": "2024-12-16T09:00:00",
+        "assessment_start": "2025-01-18T09:00:00",
+        "deadline": "2025-01-17T17:00:00",
+        "application_reminder_sent": False,
+        "reminder_date": "2025-01-10T11:00:00",
+        "assessment_deadline": "2025-01-31T17:00:00",
+        "prospectus": "",
+        "privacy_notice": "https://www.gov.uk/government/publications/local-plan-delivery-and-green-belt-reviews-expression-of-interest/funding-to-support-local-authorities-with-the-costs-of-local-plan-delivery-and-green-belt-reviews-privacy-notice",
+        "reference_contact_page_over_email": False,
+        "contact_email": "LocalPlansandGreenBeltFunding@communities.gov.uk",
+        "contact_phone": "",
+        "contact_textphone": "",
+        "support_times": "9am to 5pm",
+        "support_days": "Monday to Friday",
+        "instructions_json": {"en": None, "cy": None},
+        "feedback_link": "",
+        "project_name_field_id": "RoLhhf",
+        "application_guidance_json": {
+            "en": "<h2 class='govuk-heading govuk-heading-s'>What we'll ask you for</h2> <p class='govuk-body'>You can preview the <a href='{all_questions_url}'>full list of application questions</a>.</p>",
+            "cy": None,
+        },
+        "guidance_url": "https://www.gov.uk/government/publications/local-plan-delivery-and-green-belt-reviews-expression-of-interest",
+        "all_uploaded_documents_section_available": False,
+        "application_fields_download_available": False,
+        "display_logo_on_pdf_exports": False,
+        "mark_as_complete_enabled": True,
+        "is_expression_of_interest": False,
+        "eoi_decision_schema": {"en": {}, "cy": {}},
+        "feedback_survey_config": {
+            "has_feedback_survey": True,
```

Comment:
Is this config enabled the feedback survey ?

Reply:
Feedback survey is showing up.

Reply:
We go ahead with this config.

Reply:
We go ahead with this config.